src/server/server_family.cc

```diff
@@ -2810,6 +2810,7 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
   if (should_enter("CLUSTER")) {
     append("cluster_enabled", IsClusterEnabledOrEmulated());
     append("migration_errors_total", service_.cluster_family().MigrationsErrorsCount());
+    append("migrated_keys_total", service_.cluster_family().MigratedKeysCount());
```

Comment:
if this was the reason for atomic ints then it's not a good reason.
we have the whole system of how we gather metrics using shared nothing approach. it works well and we can extend it to cluster_family as well.